File: 06. Perulangan/D. While + Pencacah/d_while_pencacah.go
package main

import (
  "fmt"
)

func main() {
  var num, res int32
  res = 0
  for {
    _, err := fmt.Scanf("%d", &num)
    if err != nil {
      break
    }
    res += num
  }
  fmt.Println(res)
}

File: 06. Perulangan/D. While + Pencacah/d_while_pencacah.rs
use std::io;
use std::io::prelude::*;
fn main() {
    let stdin = io::stdin();
    let mut res: i32 = 0;
    for str in stdin.lock().lines() {
        let num: i32 = str.unwrap().parse().unwrap();
        res += num;
    }
    println!("{}", res);
}
